javascript/modules/client/src/internal/graphql-queries/index.ts

```diff
@@ -1,7 +1,5 @@
 export * from "./dao";
 export * from "./status";
 export * from "./balances";
-export * from "./proposal";
-export * from "./settings";
-export * from "./token";
+export * from "../../client-erc20/internal/graphql-queries/token";
```

Comment:
Exporting plugin-based queries like this on a PR called "split" might be a bit confusing